src/test/kotlin/calculator/CalculatorTest.kt

```diff
@@ -0,0 +1,62 @@
+package calculator
+
+import org.assertj.core.api.Assertions
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.MethodSource
+import java.util.AbstractMap
+import java.util.stream.Stream
+import kotlin.collections.Map.Entry
+
+class CalculatorTest {
+
+    companion object {
+        @JvmStatic
+        fun createBlankStrings(): Stream<String> {
+            return Stream.of(null, "", "   ")
+        }
+
+        @JvmStatic
+        fun createCalculation(): Stream<AbstractMap.SimpleEntry<String, Int>> {
+            return Stream.of(
+                AbstractMap.SimpleEntry("3 + 5", 8),
+                AbstractMap.SimpleEntry("3 - 5", -2),
+                AbstractMap.SimpleEntry("3 * 5", 15),
+                AbstractMap.SimpleEntry("15 / 3", 5),
+
+                AbstractMap.SimpleEntry("3 + 5 - 2", 6),
+                AbstractMap.SimpleEntry("3 - 5 * 3", -6),
+                AbstractMap.SimpleEntry("3 * 5 + 2", 17),
+                AbstractMap.SimpleEntry("15 / 3 - 5", 0),
+
+                AbstractMap.SimpleEntry("3 + 5 - 2 * 7", 42),
+                AbstractMap.SimpleEntry("3 - 5 * 3 / 2", -3),
+                AbstractMap.SimpleEntry("3 * 5 + 2 - 10", 7),
+                AbstractMap.SimpleEntry("15 / 3 - 5 + 100", 100),
+
+                AbstractMap.SimpleEntry("15 / 3 - 5 + 100 * 5", 500)
+
+            )
+        }
+    }
+
+    @ParameterizedTest
+    @MethodSource("createBlankStrings")
```

Comment:
`@NullAndEmptySource`를 사용하면 어떨까요? 아래의 글을 참고하세요.
https://www.baeldung.com/parameterized-tests-junit-5

```suggestion
    @NullAndEmptySource
```